### PR TITLE
update bootc/inference/podman.sh

### PR DESCRIPTION
**Pull request by sallyom**
NOTE: with `podman 5.0` I need to also add `--network=host` to all `podman build` commands - should be fixed with podman 5.0.1

### CODE REVIEW

File: bootc/podman_helper.sh [hunk 1]
```diff
@@ -115,14 +125,18 @@ Time for video
 }
 
 function create_disk_image {
+    storedir=$(pwd)/store
+    sudo mkdir -p ${storedir}
```
Comment (sallyom) [inline bootc/podman_helper.sh:129]: my mac did not like set to `/tmp/store` i don't know why 🤷

File: bootc/podman_helper.sh [hunk 2]
```diff
@@ -115,14 +125,18 @@ Time for video
...
+
+    XDG_RUNTIME_DIR="${XDG_RUNTIME_DIR:-/tmp}"
```
Comment (sallyom) [inline bootc/podman_helper.sh:131]: without this initialization, i get unbound variable error from Mac

File: bootc/podman_helper.sh [hunk 3]
```diff
@@ -166,6 +180,10 @@ Inspect the OCI Manigest"
 function oci_test {
     echo_color "
 Test bootable OCI image as a container"
-    exec_color "podman run --privileged --rm -it ${IMAGE} /sbin/init"
+    podman stop test-bootc
+    exec_color "podman run -d --privileged --name test-bootc --rm -it ${IMAGE} /sbin/init"
+    exec_color "podman exec -it test-bootc podman images"
+    exec_color "podman exec -it test-bootc cat /etc/redhat-release"
+    #exec_color "podman exec -t test-bootc systemctl status ${APP}"
```
[the remark this answers is not in the document]
Comment (rhatdan) [reply bootc/podman_helper.sh:187]: exec_color "podman exec -t test-bootc systemctl status ${APP} --no-pager"

File: bootc/podman_helper.sh (hunk 1 above)
Comment (rhatdan) [reply bootc/podman_helper.sh:129]: I think you want the storedir to be on the local host system on the mac.

File: bootc/podman_helper.sh [hunk 4]
```diff
@@ -38,11 +35,12 @@ function help {
 	echo -n "
 Valid options:
 
--A APP
--a ARCH
--o OS
--r REGISTRY
--v VARIANT
+-A APP (from ai-lab-recipes, options: chatbot, rag)
```
Comment (rhatdan) [inline bootc/podman_helper.sh:38]: Hopefully this list will grow quickly.

File: bootc/podman_helper.sh [hunk 5]
```diff
@@ -38,11 +35,12 @@ function help {
...
+-a ARCH (target arch to build for, default '$(uname -m)')
+-o OS (default '$(uname)')
+-r REGISTRY (default 'quay.io/rhatdan')
```
Comment (rhatdan) [inline bootc/podman_helper.sh:41]: We should probably make this a required entry.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**